src/pages/admin/AdminTicketDashboard.vue

```diff
@@ -74,3 +127,53 @@ export default {
   },
 };
 </script>
+
+<style>
+@media (min-width: 600px) {
+  .gridContainer {
+    grid-template-columns: repeat(1, 1fr);
+  }
+}
+
+@media (min-width: 900px) {
```

Comment:
FYI: tailwind has helper utilities for this:
https://tailwindcss.com/docs/adding-new-utilities#generating-responsive-variants
https://tailwindcss.com/docs/responsive-design